packages/viewer/src/components.d.ts

```diff
@@ -205,6 +205,10 @@ export namespace Components {
      * A list of the metadata keys that a scene tree search should be performed on.
      */
     metadataSearchKeys: MetadataKey[];
+    /**
+     * Indicates whether the metadata search should remove hidden items from the results.
+     */
+    metadataSearchRemoveHiddenItems: boolean;
```

Comment:
Is this feature only supported in the context of metadata search? Even in the context of a regular tree text search, shouldn't this be supported? My suggestion would be to name this something like 

💡  `filterHiddenItems` or something like thats, so it's clear to the developer that it's a setting on filtering.

or a filter object that contains something like:


```
FilterOptions: {
  text: string,
  removeHiddenItems: boolean
  } 
  ```